aspnetcore/includes/reliableWAP.md

```diff
@@ -0,0 +1 @@
+See [The Reliable Web App Pattern for .NET](https://www.youtube.com/watch?v=hNoUT9NRzDM) YouTube videos for guidance on refactoring a monolithic ASP.NET framework app using SQL Server to a modern, reliable, performant, testable, cost-efficient, and scalable ASP.NET Core app.
```

Comment:
I don't want to spook folks who are not migrating a legacy app. Can we change this to 
creating  a modern, reliable, performant, testable, cost-efficient, and scalable ASP.NET Core app.

Reply:
You're right - I think it would be best to call out to both crowds. Could we say something along the lines of:

"... for guidance on creating XXX ... whether from scratch or refactoring an existing app" ?